src/vscode/.devcontainer.json

```diff
@@ -4,13 +4,8 @@
   "service": "app",
   "shutdownAction": "none",
   "workspaceFolder": "/workspace",
-  "postCreateCommand": [
-    "./startupscript/post-startup.sh",
-    "abc",
-    "/config",
-    "${templateOption:cloud}",
-    "${templateOption:login}"
-  ],
+  "postCreateCommand":
+    "./startupscript/post-startup.sh abc /config ${templateOption:cloud} ${templateOption:login}; ./sudo-passwordless.sh abc",
```

Comment:
nit: it might be better for the `templateOption`s to be double quoted so that they can't get expanded into multiple values